docs/development.md

```diff
@@ -0,0 +1,60 @@
+Development environment
+======================
+
+The purpose of the development environment is only for running the different test and metric tools.
+
+To get started, first setup the development environment. The dev. env. is a virtual machine. Every task described is run
+from that machine.  
+
+Requirements
+-------------------
+- Vagrant 2.2.x
+    - vagrant-hostsupdater (1.1.1.160, global, optional)
+    - vagrant-vbguest (0.19.0, global)
+- Virtualbox
+
+Install
+-------------------
+
+### 1. Create virtual machine
+
+``` cd homestead ``` 
+ 
+``` composer install ```
+
+Go to root of the project (```cd ..```) 
+
+``` vagrant up ```
+
+If everything goes as planned you can develop inside the virtual machine
+
+``` vagrant ssh ```
+
+### 2. Build frontend assets:
+
+``` yarn install ```
```

Comment:
Personally, I needed to `cd` to the `code` directory in order to reach the project files on my guest.

Reply:
And on my previous installation, where I followed these steps quite closely, the after.sh script was not executed. Leaving me without a set of installed composer dependencies.

Reply:
Fixed when setup.sh runs normaly